### PR TITLE
Import script from govuk-guix

### PR DESCRIPTION
**Pull request by cbaines**
The `govuk-connect` script started in the `govuk-guix` Git repository as that included other interconnected scripts. It's not actually related to Guix though, so this Pull Request looks at moving it to this Git repository.

I've copied over the Git history, as I didn't want to loose that.

This follows on from the previous (failed) attempt at moving some more non-Guix related stuff out of `govuk-guix`: https://github.com/alphagov/govuk-guix/issues/36

### CODE REVIEW

**Comment by cbaines** [comment]
This hopefully works, but there are a few more things that need tweaking now that this isn't in `govuk-guix`.

File: govuk-connect [hunk 1]
```diff
@@ -0,0 +1,1060 @@
...
+  govuk connect sidekiq-monitoring -e integration
+EXAMPLES
+
+MACHINE_TARGET_DESCRIPTION  = <<DOCS
+  The ssh, rabbitmq and sidekiq-monitoring connection types target
+  machines.
+
+  The machine can be specified by name, for example:
+
+    govuk connect ssh -e integration #{bold('backend')}
```
Comment (issyl0) [inline govuk-connect]:
```suggestion
    govuk-connect ssh -e integration #{bold('backend')}
```

These all need updating.

File: govuk-connect [hunk 2]
```diff
@@ -0,0 +1,1060 @@
...
+
+def error(message)
+  STDERR.puts "\e[1m\e[31m#{message}\e[0m"
+end
+
+def print_ssh_username_configuration_help
+  info "The SSH username used was: #{bold(ssh_username)}"
+  info "Check this is correct, and it it isn't run the following command to set the correct username"
+  newline
+  info "  #{bold('govuk config ssh_username USERNAME')}"
```
Comment (issyl0) [inline govuk-connect]: This won't work. But luckily it also can use the `USER` envvar.

File: govuk-connect (hunk 1 above)
Comment (theseanything) [reply govuk-connect]: probably should update the same thing in the examples, and comments above as well

**Comment by theseanything** [comment]
🎉

**Pull request merged**